skietbaan_fe/src/actions/types.js

```diff
@@ -9,3 +9,17 @@ export const BASE_URL = "https://api.skietbaan.retrotest.co.za";
 export const FETCH_LEADERBOARDFILTER_DATA = 'FETCH_LEADERBOARDFILTER_DATA';
 export const FETCH_LEADERBOARDTABLE_DATA = 'FETCH_LEADERBOARDTABLE_DATA';
 export const URL = "https://api.skietbaan.retrotest.co.za"; 
+// export const URLADD = "https://api.skietbaan.retrotest.co.za/api/groups/add"; 
+// export const URLUSER = "https://api.skietbaan.retrotest.co.za/api/user"; 
+// export const URLGROUP = "https://api.skietbaan.retrotest.co.za/api/groups"; 
+export const URLADD = "http://localhost:63474/api/groups/add"; 
+export const URLUSER = "http://localhost:63474/api/user"; 
+export const URLGROUP = "http://localhost:63474/api/groups"; 
+export const UPDATENAME = 'UPDATENAME';
+export const UPDATE_GROUPNAME = 'UPDATE_GROUPNAME';
+export const CREATEGROUP = 'CREATEGROUP';
+export const GETGROUP = 'GETGROUP';
+export const PASS_ID = 'PASS_ID';
+export const GETNAME = 'GETNAME';
+
+FETCH_LEADERBOARDFILTER_DATA
```

Comment:
Pretty sure this was left here by mistake.
Please group these types into a constant object. 
So like this 
```
export const MeaningfulName = {
 UPDATENAME: 'UPDATENAME',
 UPDATE_GROUPNAME: 'UPDATE_GROUPNAME',
 CREATEGROUP: 'CREATEGROUP',
 GETGROUP: 'GETGROUP',
}
```